packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -88,12 +90,18 @@ export class MetriportMedicalApi {
    * @param options.timeout - Connection timeout in milliseconds, default 20 seconds.
```

Comment:
Should we add options.mode here?

Reply:
I intentionally left it out. This is an option only for FEs, and the only FE enabled to use our API is our Dashboard (the JWT validation is based on our Auth, so it couldn't be used by our cxs even if we wanted to).